index.js

```diff
@@ -30,8 +33,8 @@ export function zwitch(key, options) {
    * All arguments, and the context object, are passed through to the handler,
    * and it’s result is returned.
    *
-   * @param {any} [value]
-   * @returns {any}
+   * @param {...unknown} [value]
```

Comment:
I based this off of line 48 `fn.apply(this, arguments)` which implies there may be more args?
If that interpretation is incorrect and value will be a single, not a rest of many args
then this should be:

```suggestion
   * @param {unknown} [value]
```

Reply:
yep, you’re right. Multiple are allowed!